PURL-TYPES.rst

```diff
@@ -87,6 +87,25 @@ bitnami
       pkg:bitnami/wordpress@6.2.0
       pkg:bitnami/wordpress@6.2.0?arch=arm64
 
+brew
+----
+``brew`` for Homebrew-based packages:
+
+- The default repository (tap) is ``homebrew/core``.
+
+  - The tap syntax is ``https://github.com/{org}/Homebrew-{tap}``, so
+    ``homebrew/core`` corresponds to the URL ``https://github.com/homebrew/homebrew-core``.
+- The ``name`` is the formula name.
```

Comment:
**suggestion (non-blocking):** Add this within the previous list item:
```suggestion
  - Any git URL is a valid tap URL (see ``tap_url``)
- The ``name`` is the formula name.
```

This clarifies from the start, but maybe the `tap_url` bullet belongs in here?